LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Please remove this

Reply:
I planned to remove this... Will be removed